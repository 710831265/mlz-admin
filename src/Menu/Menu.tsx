import React from 'react';
import { Menu as AntdMenu } from 'antd';

const Menu = () => {
  return AntdMenu;
};
export default Menu;
