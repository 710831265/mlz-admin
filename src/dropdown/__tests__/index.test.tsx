import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import Dropdown from '..';
import Menu from '../../menu';

const TempDropdownMounter = () => (
  <Dropdown
    overlay={
      <Menu>
        <Menu.Item>
          <a target="_blank" rel="noopener noreferrer" href="http://www.alipay.com/">
            1st menu item
          </a>
        </Menu.Item>
        <Menu.Item>
          <a target="_blank" rel="noopener noreferrer" href="http://www.taobao.com/">
            2nd menu item
          </a>
        </Menu.Item>
        <Menu.Item>
          <a target="_blank" rel="noopener noreferrer" href="http://www.tmall.com/">
            3rd menu item
          </a>
        </Menu.Item>
        <Menu.Item danger>a danger item</Menu.Item>
      </Menu>
    }>
    <a className="ant-dropdown-link" onClick={(e) => e.preventDefault()}>
      Hover me
    </a>
  </Dropdown>
);

describe('ðŸ§ª Dropdown', () => {
  testMount(TempDropdownMounter);
  testSnapshot(TempDropdownMounter);
});
