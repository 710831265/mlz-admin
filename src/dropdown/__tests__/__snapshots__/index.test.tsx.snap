// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Dropdown snapshot test snapshot comparing 1`] = `"<a class=\\"ant-dropdown-trigger ant-dropdown-link\\">Hover me</a>"`;
